docs/tools.md

```diff
@@ -553,6 +553,27 @@ It prints the MKey
 ```
 
 
+## mt-parrot
+
+```
+generate deterministic metrics for each metrictank partition
+
+Usage:
+  mt-parrot [flags]
+
+Flags:
+      --artificial-metrics-interval duration   interval to send metrics (default 5s)
```

Comment:
this word is needlessly complex.
i would recommend you `s/artificial/test/` all across this code.

Reply:
6bd7c907a02afb15c2d58376e78a3a25bb94910a